ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Footer.kt

```diff
@@ -145,7 +150,9 @@ private fun Footer(
                 R.string.privacy,
             ) {
                 uriHandler.openUriOrElse(it.toString()) {
```

Comment:
We could potentially rename this function, so it's clear the `else` should only be called when there is no browser. But NABD.

Reply:
Hmm what name would you suggest? It was inspired by the stdlib's `Map.getOrElse()`, which I think is similar in spirit. ("Do this thing (get / openUri), or if that fails for whatever reason, do this other thing (orElse).")

Reply:
Well, honestly I don't like any of the namings I'm coming up with.... But maybe redefine the function to:
```
fun UriHandler.openUri(uri: String, browserNotFoundAction: () -> Unit) {
```

Alternatively, we can keep it as it is, but catch all exceptions, and pass them as a parameter to the `fallbackAction` like:
```
fun UriHandler.openUriOrElse(uri: String, fallbackAction: (Exception) -> Unit) {
```
And changing the body to catch all exceptions. 

But as I said, I'm also ok with what it is, just that it could be confused as it being called in other exceptions.

Reply:
Interesting idea! But I think my current preference is to leave it as is. The `Exception` is also kind-of an implementation detail of the function, as the same functionality could have been achieved without a try-catch, but by querying the `PackageManager` for Activities that can resolve the implicit `Intent`.  
  
It's an easy-to-reverse decision if we end up hating the name 😄

Reply:
Yeah I think that's fine 👍